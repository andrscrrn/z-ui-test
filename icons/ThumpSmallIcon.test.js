import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import ThumpSmallIcon from "./ThumpSmallIcon";

describe("ThumpSmallIcon", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<ThumpSmallIcon />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
