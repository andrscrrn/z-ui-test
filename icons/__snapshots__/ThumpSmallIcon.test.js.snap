// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThumpSmallIcon renders correctly 1`] = `
<svg
  height="28px"
  width="28px"
>
  <path
    d="M26.012,15.541 C26.012,15.541 27.285,16.026 27.285,17.500 C27.285,18.993 25.960,19.666 25.088,19.666 C25.631,19.929 26.389,20.434 26.222,21.711 C26.055,22.987 25.121,23.451 24.199,23.727 C24.795,24.085 25.229,25.077 24.897,25.841 C24.450,26.868 23.271,28.000 20.625,28.000 C17.423,28.000 8.615,28.000 8.264,28.000 C7.327,28.000 6.631,27.583 6.485,27.070 C6.485,26.017 6.485,13.001 6.485,13.001 C6.485,13.001 6.485,12.549 6.555,12.431 C9.852,7.701 11.022,6.353 10.827,3.615 C10.632,0.878 11.190,0.124 12.518,-0.011 C13.846,-0.145 16.341,0.561 16.779,5.042 C16.968,6.980 15.547,10.798 15.547,10.798 C15.547,10.798 24.610,10.804 25.490,10.804 C26.898,10.804 28.000,11.391 28.000,13.262 C28.000,15.015 26.012,15.541 26.012,15.541 ZM0.781,28.000 L-0.000,28.000 L-0.000,11.531 L0.781,11.531 L4.844,11.531 L5.625,11.531 L5.625,28.000 L4.844,28.000 L0.781,28.000 Z"
    fill="rgb(255, 255, 255)"
    fillRule="evenodd"
  />
</svg>
`;
